satellite/analytics/hubspot.go

```diff
@@ -0,0 +1,167 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package analytics
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/spacemonkeygo/monkit/v3"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"storj.io/common/sync2"
+)
+
+var mon = monkit.Package()
+
+// HubSpotConfig is a configuration struct for Concurrent Sending of Events
+type HubSpotConfig struct {
+	APIKey          string `help:"hubspot api key" default:""`
+	ChannelSize     int    `help:"the number of events that can be in the queue before dropping" default:""`
```

Comment:
these two configs should have a default value. Let's do 10000 for ChannelSize and 25 for ConcurrentSends